pkg/operator/controller/controller.go

```diff
@@ -168,11 +170,21 @@ func (r *reconciler) enforceEffectiveIngressDomain(ci *operatorv1.IngressControl
 	updated := ci.DeepCopy()
 	switch {
 	case len(ci.Spec.Domain) > 0:
+		unique, err := r.ensureUniqueDomain(updated)
+		if err != nil {
+			return err
+		}
+		if !unique {
+			// TODO: Create DNSConflict status condition when the following PR merges:
+			//       https://github.com/openshift/cluster-ingress-operator/pull/174
+			log.Info("domain conflict, not setting domain in status for clusteringress: ", "namespace", updated.Namespace, "name", updated.Name)
```

Comment:
Any concern about log spam? What about an event in this case prior to our use of status?

Reply:
Since the ingresscontroller completes reconciliation, I only see one log message.